packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -440,7 +440,10 @@ class HorizontalOverflow extends Foundation<
     private isOverflow(): boolean {
         const availableWidth: number = this.getAvailableWidth();
         const itemWidths: number[] = this.getItemWidths();
-        const totalItemWidth: number = itemWidths.reduce((a: number, b: number) => a + b);
+        const totalItemWidth: number = itemWidths.reduce(
+            (a: number, b: number) => a + b,
+            0
+        );
```

Comment:
Should we add a test for this?

Reply:
Not sure how to fake it "for realz", but I could add a direct function test with staged values.

Reply:
I wouldn't worry about it here as now we handle the empty array. I guess we could say, "empty array shouldn't throw"?

Reply:
added